docs/guides/runtime-options-overview.mdx

```diff
@@ -135,11 +137,16 @@ Scroll to see the full table.
 |                      | num_randomizations       |                         |  `auto`/ integer >= 1                                                                        | `'auto'`                    |                    |
 |                      | shots_per_randomization  |                         |   `auto`/ Integer no larger than `backend.max_shots`.                                        | `'auto'`                    |                    |
 |                      | strategy                 |                         | `'active'`<br />`'active-circuit'`<br />`'active-accum'`<br />`'all'`                        | `'active-accum'`            |                    |
+| experimental         |                          |                         |                           |                        |                    |
+|                      | execution_path           |                         | `<new_flag>` *([see note](#experimental-option-note))*               | None                   |                    |
 
   </TabItem>
 </Tabs>
 
-
+<span id="experimental-option-note"></span>
+<Admonition type="note">
+Setting `<new flag>` enables the next-generation execution path. This is currently experimental and is only available on `ibm_torino`.
```

Comment:
It sounds like it's experimental and temporary.  "currently" can imply that we're going to change it to be non-experimental in the future, so I'd remove that word.